InternalDocs/garbage_collector.md

```diff
@@ -448,31 +448,23 @@ The content of these generations can be examined using the
 `gc.get_objects(generation=NUM)` function and collections can be triggered
 specifically in a generation by calling `gc.collect(generation=NUM)`.
 
-```python
+```pycon
 >>> import gc
 >>> class MyObj:
 ...     pass
 ...
-
-# Move everything to the old generation so it's easier to inspect
-# the young generation.
-
+>>> # Move everything to the old generation so it's easier to inspect the young generation.
```

Comment:
This line is too long.  Ditto similar case below.  Just don't reformat comments.